bentoml/_internal/frameworks/keras.py

```diff
@@ -188,8 +182,9 @@ def save(
     model: "keras.Model",
     *,
     save_format: t.Optional[str] = "tf",
-    custom_objects: t.Optional[t.Dict[str, t.Any]] = None,
     store_as_json_and_weights: t.Optional[bool] = False,
+    labels: t.Optional[t.Dict[str, str]] = None,
+    custom_objects: t.Optional[t.Dict[str, t.Any]] = None,
```

Comment:
should we differentiate Keras custom_objects with our model store custom_objects?

Reply:
In my test we can mix them together. So saving keras custom_objects with other custom_objects won't affect how model is loaded.

We can also add a `keras_custom_objects` argument, but do you think it's a little bit cumbersome? @parano